### PR TITLE
Add mime filter

### PR DESCRIPTION
**Pull request by hibeekaey**

### CODE REVIEW

File: www/fileChooser.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module.exports = {
-    open: function (success, failure) {
-        cordova.exec(success, failure, "FileChooser", "open", []);
+    open: function (filter, success, failure) {
```
Comment (ihadeed) [inline www/fileChooser.js:2]: can we maintain backwards compatibility here? 

```
if (typeof filter === 'function') {
  failure = success;
  success = filter;
  filter= [];
}
```

**Review by ihadeed — CHANGES_REQUESTED** [review]

File: www/fileChooser.js (hunk 1 above)
Comment (hibeekaey) [reply www/fileChooser.js:2]: Yes, I've worked on it. I also cleaned up the README.

**Comment by ansarikhurshid786** [comment]
is filter working?

**Comment by hibeekaey** [comment]
Yes, filter is working.

**Comment by ansarikhurshid786** [comment]
How to add mime type ? can you give me one example please because I can't find it in documentation or anywhere else

**Comment by hibeekaey** [comment]
You can find the updated repo and documentation here, https://github.com/hibeekaey/cordova-filechooser

**Pull request merged**